### DISCUSSION TITLE
Should lit babel docs remove @babel/plugin-proposal-class-properties?

### DISCUSSION THREAD

**Discussion by yringler** [lit/lit#4416] [General]
[Lit babel docs](https://lit.dev/docs/tools/publishing/#compiling-with-babel) recommend adding @babel/plugin-proposal-class-properties, a [deprecated plugin](https://www.npmjs.com/package/@babel/plugin-proposal-class-properties), and I don't see explanation why.
Should that recommendation be removed, or updated?

**Comment by augustjk** [comment]
Thanks, that's a good callout. We removed it from from the [decorators with babel section](https://lit.dev/docs/components/decorators/#decorators-babel) but didn't catch it here. We should update that section for publishing too.

**Reply by augustjk** [reply]
Filed issue in lit.dev repo: https://github.com/lit/lit.dev/issues/1275
